src/java.desktop/unix/classes/sun/print/UnixPrintJob.java

```diff
@@ -525,8 +524,6 @@ public void print(Doc doc, PrintRequestAttributeSet attributes)
 
         // now spool the print data.
         PrinterOpener po = new PrinterOpener();
-        @SuppressWarnings("removal")
-        var dummy = java.security.AccessController.doPrivileged(po);
```

Comment:
You are no longer running the action

Reply:
I'm not sure if I understood it correctly: Did you mean remove `implements java.security.PrivilegedAction<Object>` from PrinterOpener class ?

Reply:
Well, that too. But the main point is WHERE is the code now that calls PrinterOpener.run() ?
The thing that does the actual work. So far as I can see you've deleted the call and done nothing to replace it.

Reply:
So 2 things to do that I can see.
1) after line 526 call po.run();
2) Go to the definition of PrinterOpener and remove implements java.security.PrivilegedAction<Object>

Anything more would be too much refactoring.

Same idea below for PrinterSpooler.

How were you testing this ?

Reply:
> after line 526 call po.run();
> Go to the definition of PrinterOpener and remove implements java.security.PrivilegedAction
> Anything more would be too much refactoring.

Makes sense. Thank you for clarifying.

> How were you testing this ?

Ran CI tests - clientlibs, wayland jobs for these unix file changes as of now. I believe to test this properly we need to run manual printer tests.